Gemfile

```diff
@@ -108,6 +108,8 @@ gem 'icalendar', '~> 2.2.2'
 
 gem 'rubyzip', '~> 1.2.2'
 
+gem "ce_diploma", :git => "git://github.com/sis-berkeley-edu/ce_diploma.git"
```

Comment:
@developish - I will be removing this, because we're relying on a Campus Solutions API for this functionality now. This gem is just a fall back measure.